magentoerpconnect/magento_model.py

```diff
@@ -186,17 +187,23 @@ def check_magento_structure(self):
 
     @api.multi
     def synchronize_metadata(self):
-        session = ConnectorSession(self.env.cr, self.env.uid,
-                                   context=self.env.context)
-        for backend in self:
-            for model in ('magento.website',
-                          'magento.store',
-                          'magento.storeview'):
-                # import directly, do not delay because this
-                # is a fast operation, a direct return is fine
-                # and it is simpler to import them sequentially
-                import_batch(session, model, backend.id)
-        return True
+        try:
+            session = ConnectorSession(self.env.cr, self.env.uid,
+                                       context=self.env.context)
```

Comment:
Could you take the opportunity of this diff to improve that line to `ConnectorSession.from_env(self.env)`?